h2o-algos/src/main/java/hex/generic/GenericModel.java

```diff
@@ -70,7 +70,7 @@ private static MojoModel reconstructMojo(ByteVec mojoBytes) {
             case WordEmbedding:
                 return unsupportedMetricsBuilder();
             case CoxPH:
-                return unsupportedMetricsBuilder();
+                return new ModelMetricsRegressionCoxPH.MetricBuilderRegressionCoxPH("start", "stop", false, new String[0]);
```

Comment:
Missing `GenericModelTest` tests. Re-using other tests is fine. There are tests for mojo "downloadability" and scoring tests against Java.

Reply:
Downloadabily is in place..

Reply:
...and scoring too